job_templates/puppet_agent_disable_-_ssh_default.erb

```diff
@@ -0,0 +1,17 @@
+<%#
+name: Puppet Agent Disable - SSH Default
+model: JobTemplate
+job_category: Puppet
+description_format: Disable Puppet agent
+snippet: false
+template_inputs:
+- name: comment
+  required: false
+  input_type: user
+  description: Reason for disabling the Puppet agent
+  advanced: false
+provider_type: SSH
+kind: job_template
+%>
```

Comment:
Ideally add - before % to avoid extra line, I suppose you exported this from Foreman, so we may need to update oir exporter